packages/components/src/OverlayTrigger/overlay.js

```diff
@@ -0,0 +1,109 @@
+const OVERLAY_PLACEMENT_TOP = 'top';
+const OVERLAY_PLACEMENT_BOTTOM = 'bottom';
+const BODY = 'BODY';
+const OVERLAY_CONTAINER_CLASS_NAME = 'tc-dropdown-container';
+const REACT_BOOSTRAP_OVERLAY_CLASS_NAME = 'popover';
+
+/**
+ * getOverlayElement - return the overlay element
+ *
+ * @param  {Element} innerElement the root element of the action
+ * @return {Element}              return the overlay component
+ */
+export function getOverlayElement(innerElement) {
```

Comment:
Native API is always better code because less code :)
https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Can totally delete the function and directly use something of this smell.
```javascript
innerElement.closest(`.${REACT_BOOSTRAP_OVERLAY_CLASS_NAME}`); 
```
Need to check if compliant with browsers range supported in our projects (I don't know the real exacts versions), there is still a polyfill for some oldest version (not before IE9, etc. however).

Reply:
nice but no supprt on IE :)

Reply:
> For browsers that do not support Element.closest(), but carry support for element.matches() (or a prefixed equivalent, meaning IE9+), a polyfill exists

Reply:
need to implement babel-polyfill to implement that.